components/sync15/src/migrate_state.rs

```diff
@@ -123,6 +123,7 @@ mod tests {
         get_state_with_engine_changes_and_declined(changes, "")
     }
 
+    #[allow(clippy::unnecessary_wraps)]
```

Comment:
nit: I like the way you had comments on other instances of `allow`, please consider adding one here as well.

Reply:
done.